src/components/tls-checker/index.ts

```diff
@@ -24,15 +24,21 @@
 
 import sslChecker from 'ssl-checker'
```

Comment:
- [ ] please also add a test in `src/components/tls-checker/index.test.ts`
- [ ] please update the documentation in `docs/src/pages/guides/tls-checkers.md`

Reply:
Added and updated.